### DISCUSSION TITLE
クローズドベータが終了しても議論を消さないでほしい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#11] [要望・アイデア・不具合]
これは私のわがままですが、一生懸命要望を出したので、クローズドベータが終了しても議論を残してほしいですｗ

**Comment by kskwtnk** [comment]
@uhooi 

いえいえとんでもないです！

ここまでたくさんDiscussionを作成していただけると想定しておらず、とても嬉しいです！

議論は消さず、このリポジトリをそのまま残すか、本運用としてIncrementsのOrganization配下にTransferするかのつもりでいます！

**Reply by uhooi** [reply]
わーい！
ありがとうございます 🙏

**Comment by blono** [comment]
このリポジトリには権利関係についての定めが特にないため、残すにしても公開する場合には個別的な同意は必要かと思われます。
私のDiscussionについても、公開してくださって問題ありません。

**Reply by kskwtnk** [reply]
@blono 

本当ですね、ありがとうございます……！
当初のイメージでは、このリポジトリとは別にパブリックなリポジトリを作成してそちらを本運用として使っていくつもりで規約を作成していたのですが、こういうシーンが訪れる想定が出来ていませんでした

弊社法務にも相談して、同意をとる方法や保管の仕方を整理します！ありがとうございます！

**Reply by kskwtnk** [reply]
@blono 

アドバイスありがとうございました、以下のように進めていく予定です！

1. 立てたDiscussion、コメントした内容、リアクションを公開しても良いか確認するGoogleフォームを作成し、クローズ時にみなさまに展開
1. 回答データを弊社管轄のGoogleドライブに保管
